### DISCUSSION TITLE
Are the RSF and GradientBoostingSurvivalAnalysis, Regression or Classification Models?

### DISCUSSION THREAD

**Discussion by shreybothra** [sebp/scikit-survival#276] [Q&A]
Exactly what the title says. I am very new to ML, so please excuse such a basic doubt.

**Comment by sebp** [comment]
I prefer to consider them as separate class of models, namely time-to-event models. But you could also make the case that they are regression models, but where the training/test data has missing information (the censored event times).

**Comment by sebp** [comment]
I prefer to consider them as separate class of models, namely 
time-to-event models. But you could also make the case that they are 
regression models, but trained on data with missing information (the 
censored event times).
